packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -78,6 +78,10 @@ function getStyles(theme: GrafanaTheme2) {
         pointer-events: none;
       }
 
+      &[data-popper-interactive='true'] {
```

Comment:
If we target the tooltip, this will affect every tooltip in Grafana. The tooltip by default doesn't have any width constraint, so I'd suggest adding this property where the constraint is. In this case [in the panel description component](https://github.com/grafana/grafana/blob/32a26d87a47c1b6fe408310297d272ca3ad67e1b/public/sass/components/_panel_header.scss#L120-L121).